src/components/UserList/UserList.tsx

```diff
@@ -82,95 +75,93 @@ export function UserList() {
     setPage(0);
   };
 
-  const isSelected = (id: number) => {
-    return selected.indexOf(id) !== -1;
+  const isSelected = (row: Data) => {
+    return (
+      selected.filter((item) => {
+        return item.id === row.id;
+      }).length > 0
+    );
   };
```

Comment:
i think it will be better if you store only id's of selected rows

Reply:
if I only stored ids, the toolbar wouldn't know what to display because user types for users selected on previous pages would be lost